src/general/base_stage/HexEditorComponentBase.cs

```diff
@@ -31,6 +31,11 @@ public abstract class
     [Export]
     public NodePath IslandErrorPath = null!;
 
+    /// <summary>
+    ///   The <see cref="MicrobeCamera"/> used by this editor.
+    /// </summary>
+    public MicrobeCamera? Camera;
```

Comment:
If this must be made public, could you at least make this a property without a public setter? That way this class state cannot be destroyed by external code swapping out the camera object.